Installers/Ubuntu-Install.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+echo Please enter your password to continue...
+
+# Install the following dependencies from repositories without asking questions, git-core can be removed afterwords as it's only used to initialise a local git repository
+echo Installing dependencies
+sudo apt-get -y install git-core libusb-1.0.0 libudev-dev python3 cython3 python3-pip python3-pil python3-setuptools
+
+cd /tmp
+pip3 install hidapi click
```

Comment:
click is available as a debian package, `python3-click`. Setuptools would install it when using `setup.py` anyway.